authors/apps/articles/renderers.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.renderers import JSONRenderer
```

Comment:
@Philipotieno  Remove this file too. It is causing the coverage drop and it's not needed. I think it was left in your branch during rebasing

Reply:
Removed it